cipd_packages/device_doctor/lib/src/android_device.dart

```diff
@@ -380,6 +380,19 @@ class AndroidDevice implements Device {
 
   @override
   Future<void> recover() async {
+    // Ensure device is rebooted after deleted packages.
+    List<String> delete3PPackageArgs = [
```

Comment:
Do we need to clean the application cache as well? go/flutter-luci-playbook#android-install-failed-insufficient-storage